client/containers/DashboardReviews/DashboardReviews.js

```diff
@@ -28,7 +28,11 @@ const DashboardReviews = (props) => {
 		return pub.reviews.length;
 	});
 	return (
-		<DashboardFrame className="dashboard-reviews-container" title="Reviews">
+		<DashboardFrame
+			className="dashboard-reviews-container"
+			title="Reviews"
+			details="Reviews allow members of your Community to request feedback on their Pubs and release them to the world."
+		>
```

Comment:
Each of the other descriptions is written in active tense while this one is more of a description. Here's a really bad stab at it: "Manage requests for feedback created by members of your Community"?